applauncher-img/diskBackground.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="100" height="100" viewBox="0 0 100 100"
```

Comment:
Should you delete the ones in images/ ?

Reply:
I think that for now it makes sense to keep this indeed here too as it is also used by the `BluetoothAgent` too.

Reply:
Okay. Anyway I agree with "I actually think that it might be better to nuke the diskBackground.svgs and basically revert this commit: 15c3e53. This will worsen support on tetra, but it has broken watchfaces anyway (The masked spartan for example). And as discussed with @FlorentRevest before, it is not a good design approach to have a worse code base because one watch is misbehaving."